server/service/osquery_utils/queries.go

```diff
@@ -1201,6 +1201,18 @@ func GetDetailQueries(fleetConfig config.FleetConfig, features *fleet.Features)
 		}
 	}
```

Comment:
Let's remove the `FLEET_DANGEROUS_REPLACE_ ` functionality. What we are adding is better.